src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -447,7 +447,7 @@ public function getProtocolVersion()
      * If the status text is null it will be automatically populated for the known
      * status codes and left empty otherwise.
      *
-     * @return Response
+     * @return $this
```

Comment:
>For PHPStorm $this gives way better DX so it seems (opposed to self).

How that? We use `self` in many projects for some time now and it works pretty well.